sequencer/src/main.rs

```diff
@@ -18,10 +18,10 @@ async fn main() -> anyhow::Result<()> {
     setup_logging();
     setup_backtrace();
 
-    tracing::warn!("sequencer starting up");
+    tracing::info!("sequencer starting up");
     let opt = Options::parse();
+    tracing::info!("options: {:?}", opt);
```

Comment:
For this one I think `warn!` does make sense, as we would almost always want this to show up in the logs even if we are trying to hide other INFO level messages.

Reply:
Ok I'll change it back

Reply:
https://github.com/EspressoSystems/espresso-sequencer/pull/1409/commits/69a8b5fd1619faa1d41802f07de8eb753ddc1634